docs/architecture/adr-030-authz-module.md

```diff
@@ -103,89 +121,81 @@ using the `Authorization` interface with no need to change the underlying
 
 ```proto
 service Msg {
-  // GrantAuthorization grants the provided authorization to the grantee on the granter's
+  // Grant grants the provided authorization to the grantee on the granter's
   // account with the provided expiration time.
-  rpc GrantAuthorization(MsgGrantAuthorization) returns (MsgGrantAuthorizationResponse);
+  rpc Grant(MsgGrant) returns (MsgGrantResponse);
 
-  // ExecAuthorized attempts to execute the provided messages using
+  // Exec attempts to execute the provided messages using
   // authorizations granted to the grantee. Each message should have only
   // one signer corresponding to the granter of the authorization.
-  // The grantee signing this message must have an authorization from the granter.
-  rpc ExecAuthorized(MsgExecAuthorized) returns (MsgExecAuthorizedResponse)
+  rpc Exec(MsgExec) returns (MsgExecResponse);
 
-
-  // RevokeAuthorization revokes any authorization corresponding to the provided method name on the
+  // Revoke revokes any authorization corresponding to the provided method name on the
   // granter's account that has been granted to the grantee.
-  rpc RevokeAuthorization(MsgRevokeAuthorization) returns (MsgRevokeAuthorizationResponse);
+  rpc Revoke(MsgRevoke) returns (MsgRevokeResponse);
 }
 
-message MsgGrantAuthorization{
+message MsgGrant {
   string granter = 1;
   string grantee = 2;
-  google.protobuf.Any authorization = 3 [(cosmos_proto.accepts_interface) = "Authorization"];
-  google.protobuf.Timestamp expiration = 4;
+
+  google.protobuf.Any       authorization = 3 [(cosmos_proto.accepts_interface) = "Authorization"];
+  google.protobuf.Timestamp expiration    = 4 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
 }
```

Comment:
That's maybe a comment for #9042, but putting is here. What do you think of simplifying to:
```
message MsgGrant {
  string granter = 1;
  string grantee = 2;
  Grant grant = 3;
}
```

Reply:
It makes sense. @aaronc  what do you think, shall we do a quick update?

Reply:
I will do the change in the meantime in a new PR - if nobody will be against, I'm for it.

Reply:
In the interest of time, I have updated the type definition here, and created a PR to update the package.